tensorflow/compiler/xla/service/gpu/horizontal_loop_fusion.cc

```diff
@@ -67,6 +67,25 @@ PrimitiveType GetUniqueOutputTypeOfFusion(const HloInstruction& instr) {
   return first_output_type;
 }
 
+size_t GetInstrCountOfFusible(const HloInstruction& instr) {
```

Comment:
Is it possible to reduce the duplication with the same function in the other file?